src/prisma/generator/templates/client.py.jinja

```diff
@@ -347,7 +347,7 @@ class Prisma:
         }
 
     def _make_sqlite_url(self, url: str, *, relative_to: Path = SCHEMA_PATH.parent) -> str:
-        url_path = url.lstrip('file:').lstrip('sqlite:')
+        url_path = url.lstrip('file').lstrip('sqlite')[1:]
```

Comment:
I think the best way to fix this would be to just replace `.lstrip` with `.removeprefix` but since `removeprefix` was only added in `3.9` we'll have to define our own function that does this, e.g.

```py
def removeprefix(string: str, prefix: str) -> str:
    if string.startswith(prefix):
        return string[len(prefix):]
    return string
```

Could you add this function to `_compat.py` and use it here please? :)